docs/react-samples/src/App.tsx

```diff
@@ -33,25 +33,26 @@ function App() {
         onChange={(e) => setAccessToken(e.target.value)}
       />
       <button
-        disabled={accessToken.trim() === ""}
+        disabled={accessToken.trim() === ''}
         onClick={() => {
           store.init({webexConfig, access_token: accessToken}).then(() => {
             setIsSdkReady(true);
           });
         }}
-      >Init Widgets</button>
-      {
-        isSdkReady && (
-          <>
-            <StationLogin  onLogin={onLogin} onLogout={onLogout} />
-            {
-              isLoggedIn && <UserState />
-            }
-          </>
-        )
-      }
+      >
+        Init Widgets
+      </button>
+      {isSdkReady && (
+        <>
+          <StationLogin onLogin={onLogin} onLogout={onLogout} />
+          {isLoggedIn && <UserState />}
+          {isLoggedIn && <IncomingTask />}
+          {isLoggedIn && <TaskList />}
+        </>
+      )}
     </>
   );
 }
-
+// @ts-ignore
+window.store = store;
```

Comment:
This shouldn't be needed right? Are we going to ask developers to set store globally this way?

Reply:
This might come handy while debugging. But this is not required.